Deleting Konvoy 1.x references (#4393)
mesosphere/dcos-docs-site: pages/dkp/kaptain/2.0.0/install/index.md

enterprise: false
---

> <p class="message--important"><strong>IMPORTANT: </strong>Refer to the <a href="../fresh-install">Kaptain fresh install instructions</a>, if you already have Kaptain and want to move from 1.x to 2.x. Keep in mind. that it is not possible to migrate your data at this point.</p>

Reviewer: ```suggestion
<p class="message--important"><strong>IMPORTANT: </strong>Refer to the <a href="../fresh-install">Kaptain fresh install instructions</a>, if you already have Kaptain and want to move from 1.x to 2.x. Note that it is not possible to migrate your data.</p>
```

Reviewer: At this point -- might infer that we will someday provide the ability to migrate data, so best not to say that.
